cmd/network/set_chain_context.go

```diff
@@ -1,30 +1,59 @@
 package network
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
 
 	cliConfig "github.com/oasisprotocol/cli/config"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+)
+
+var (
+	chainContext string
+
+	setChainContextCmd = &cobra.Command{
+		Use:   "set-chain-context <name>",
```

Comment:
The chain context is optional, but it's still there. Check out how we do it in the [`oasis account withdraw`](https://github.com/oasisprotocol/cli/blob/master/cmd/account/withdraw.go#L22) command.

```suggestion
		Use:   "set-chain-context <name> [chain-context]",
```